hugegraph-rpc/pom.xml

```diff
@@ -3,49 +3,30 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.baidu.hugegraph</groupId>
+        <artifactId>hugegraph-commons</artifactId>
+        <version>1.8.10</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
 
-    <groupId>com.baidu.hugegraph</groupId>
     <artifactId>hugegraph-rpc</artifactId>
     <version>1.0.1</version>
 
     <name>hugegraph-rpc</name>
     <url>https://github.com/hugegraph/hugegraph-rpc</url>
-    <description>
-        HugeGraph Database RPC component
-    </description>
-
-    <parent>
-        <groupId>org.sonatype.oss</groupId>
-        <artifactId>oss-parent</artifactId>
-        <version>7</version>
-    </parent>
-
-    <licenses>
-        <license>
-            <name>The Apache Software License, Version 2.0</name>
-            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
-    <scm>
-        <url>https://github.com/hugegraph/hugegraph-rpc</url>
-        <connection>https://github.com/hugegraph/hugegraph-rpc</connection>
-        <developerConnection>https://github.com/hugegraph/hugegraph-rpc</developerConnection>
-    </scm>
+    <description>HugeGraph Database RPC component</description>
```

Comment:
need to update https://github.com/hugegraph/hugegraph-rpc link?

Reply:
like change to `https://github.com/hugegraph/hugegraph-commons/hugegraph-rpc` ?

Reply:
it's ok to just delete the url option